purchase_request_to_rfq/wizard/purchase_request_line_make_purchase_order.py

```diff
@@ -250,10 +251,10 @@ def make_purchase_order(self):
             # The onchange quantity is altering the scheduled date of the PO
             # lines. We do not want that:
             po_line.date_planned = item.line_id.date_required
-            res.append(purchase.id)
+            res.append(self.purchase_order_id.id)
```

Comment:
use  `res = self.item_ids.mapped('purchase_order_id').ids` outside the `for` loop.

Reply:
the items don't have purchase_order_if plus why going through the item again. I keep it this way